Fixed Swizzling crash(#533)

Kevin-007 opened the pull request:
### Summary
Fixed Swizzling crash(#533) 
Add flag to prevent recurring calls
Remove delay swizzle to allow risk for skeletonLayoutSubviews() infinite loop calls

### Requirements (place an `x` in each of the `[ ]`)
* [x] I've read and understood the [Contributing guidelines](https://github.com/Juanpe/SkeletonView/blob/main/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://github.com/Juanpe/SkeletonView/blob/main/CODE_OF_CONDUCT.md).

kikiwora commented:
It seems this change was reverted.
May we know why?

I am experiencing the same crash due to infinite recursion of skeletonLayoutSubviews() due to swizzling

Kevin-007 commented:
> It seems this change was reverted. May we know why?
- I have fork it  and manage it as a internal library. Because the fix idea have changes and it works so far.
The fixed solutions as below: 
Removed  all unSwizzle method call:
```
        unSwizzleLayoutSubviews()
        unSwizzleTraitCollectionDidChange()
```
Updated all swizzle methods as below:
```swift
        DispatchQueue.once(token: "UIView.SkeletonView.swizzleLayoutSubviews") {
            swizzle(selector: #selector(UIView.layoutSubviews),
                    with: #selector(UIView.skeletonLayoutSubviews),
                    inClass: UIView.self,
                    usingClass: UIView.self)
            self.layoutSkeletonIfNeeded()
        }
```
```swift
        DispatchQueue.once(token: "UIView.SkeletonView.swizzleTraitCollectionDidChange") {
            swizzle(selector: #selector(UIView.traitCollectionDidChange(_:)),
                    with: #selector(UIView.skeletonTraitCollectionDidChange(_:)),
                    inClass: UIView.self,
                    usingClass: UIView.self)
        }
```
The root cause is swizzle by delay 0.01,  and swizzle and unSwizzle frequently. 
Hope it help for u, thank you.
@kikiwora
